iocore/net/UnixNetVConnection.cc

```diff
@@ -1273,14 +1273,21 @@ UnixNetVConnection::connectUp(EThread *t, int fd)
     con.is_bound     = true;
   }
 
+  if (check_emergency_throttle(con)) {
+    // `con' could be closed if there is hyper emergency
+    if (con.fd == NO_FD) {
+      NET_SUM_GLOBAL_DYN_STAT(net_connections_currently_open_stat, -1);
```

Comment:
Please comment here that we need to decrement the stat because ``close_UnixNetVConnection`` only decrements with a valid connection descriptor.